### DISCUSSION TITLE
village distance village list on the right

### DISCUSSION THREAD

**Discussion by tittelityy** [adipiciu/Travian-scripts#25] [Ideas]
Pelase add village distance to the village list on the right side. Just like at the profile village list.

**Comment by adipiciu** [comment]
You can change 2 lines:

Change line
`vilT.appendChild($ee('THEAD',$em('TR',[$c(hideP),$c($gc('boxTitle',$g('sidebarBoxVillagelist'))[0].innerHTML.onlyText(),[['colspan',3],['style','font-weight:bold;']])])));`
to
`vilT.appendChild($ee('THEAD',$em('TR',[$c(hideP),$c($gc('boxTitle',$g('sidebarBoxVillagelist'))[0].innerHTML.onlyText(),[['colspan',4],['style','font-weight:bold;']]),$c('&lt;-&gt;')])));`

and this line
`vilB.appendChild($em('TR',[$c('&#x25CF;',[['class',cl]]),$c($ee('DIV',vLink[i])),RB.Setup[38] != 0 ? $c($a(printCoords(villages_id[i]),[['href',linkVSwitch[i]]])) : '',$c(addDorf12Links(linkVSwitch[i],0),[['style','width:40px;']]),$c(addARLinks(villages_id[i],0))]));`
to 
`vilB.appendChild($em('TR',[$c('&#x25CF;',[['class',cl]]),$c($ee('DIV',vLink[i])),RB.Setup[38] != 0 ? $c($a(printCoords(villages_id[i]),[['href',linkVSwitch[i]]])) : '',$c(addDorf12Links(linkVSwitch[i],0),[['style','width:40px;']]),$c(addARLinks(villages_id[i],0)),$c(parseFloat(calcDistance( villages_id[i], village_aid ).toFixed(1)))]));`
